From c4b98637b9007044f1211422a87b08fcbd7d7129 Mon Sep 17 00:00:00 2001
From: Kent Fredric <kentnl@gentoo.org>
Date: Wed, 5 Feb 2020 22:56:02 +1300
Subject: Renormalize Cargo.toml

Upstream somehow side-stepped using cargo for publishing, which
yielded a broken config.

https://github.com/bluss/indexmap/issues/116
---
 Cargo.toml | 97 +++++++++++++++++++++++++++---------------------------
 1 file changed, 49 insertions(+), 48 deletions(-)

diff --git a/Cargo.toml b/Cargo.toml
index 166a7f8..de13883 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -1,65 +1,66 @@
+# THIS FILE IS AUTOMATICALLY GENERATED BY CARGO
+#
+# When uploading crates to the registry Cargo will automatically
+# "normalize" Cargo.toml files for maximal compatibility
+# with all versions of Cargo and also rewrite `path` dependencies
+# to registry (e.g., crates.io) dependencies
+#
+# If you believe there's an error in this file please file an
+# issue against the rust-lang/cargo repository. If you're
+# editing this file be aware that the upstream Cargo.toml
+# will likely look very different (and much more reasonable)
+
 [package]
 name = "indexmap"
 version = "1.3.1"
-authors = [
-"bluss",
-"Josh Stone <cuviper@gmail.com>"
-]
+authors = ["bluss", "Josh Stone <cuviper@gmail.com>"]
+build = "build.rs"
+description = "A hash table with consistent order and fast iteration.\n\nThe indexmap is a hash table where the iteration order of the key-value\npairs is independent of the hash values of the keys. It has the usual\nhash table functionality, it preserves insertion order except after\nremovals, and it allows lookup of its elements by either hash table key\nor numerical index. A corresponding hash set type is also provided.\n\nThis crate was initially published under the name ordermap, but it was renamed to\nindexmap.\n"
 documentation = "https://docs.rs/indexmap/"
-repository = "https://github.com/bluss/indexmap"
-license = "Apache-2.0/MIT"
-description = """
-A hash table with consistent order and fast iteration.
-
-The indexmap is a hash table where the iteration order of the key-value
-pairs is independent of the hash values of the keys. It has the usual
-hash table functionality, it preserves insertion order except after
-removals, and it allows lookup of its elements by either hash table key
-or numerical index. A corresponding hash set type is also provided.
-
-This crate was initially published under the name ordermap, but it was renamed to
-indexmap.
-"""
-
 keywords = ["hashmap", "no_std"]
 categories = ["data-structures", "no-std"]
+license = "Apache-2.0/MIT"
+repository = "https://github.com/bluss/indexmap"
+[package.metadata.docs.rs]
+features = ["serde-1", "rayon"]
 
-build = "build.rs"
+[package.metadata.release]
+no-dev-version = true
+tag-name = "{{version}}"
+[profile.bench]
+debug = true
 
 [lib]
 bench = false
+[dependencies.rayon]
+version = "1.0"
+optional = true
 
-[build-dependencies]
-autocfg = "1"
-[dependencies]
-serde = { version = "1.0", optional = true }
-rayon = { version = "1.0", optional = true }
+[dependencies.serde]
+version = "1.0"
+optional = true
+[dev-dependencies.fnv]
+version = "1.0"
 
-[dev-dependencies]
-itertools = "0.8"
-rand = "0.6"
-quickcheck = { version = "0.8", default-features = false }
-fnv = "1.0"
-lazy_static = "1.3"
-serde_test = "1.0.99"
+[dev-dependencies.itertools]
+version = "0.8"
 
-[features]
-# Serialization with serde 1.0
-serde-1 = ["serde"]
-
-# for testing only, of course
-test_low_transition_point = []
-test_debug = []
+[dev-dependencies.lazy_static]
+version = "1.3"
 
-[profile.bench]
-debug = true
+[dev-dependencies.quickcheck]
+version = "0.8"
+default-features = false
 
-[package.metadata.release]
-no-dev-version = true
-tag-name = "{{version}}"
+[dev-dependencies.rand]
+version = "0.6"
 
-[package.metadata.docs.rs]
-features = ["serde-1", "rayon"]
+[dev-dependencies.serde_test]
+version = "1.0.99"
+[build-dependencies.autocfg]
+version = "1"
 
-[workspace]
-members = ["test-nostd"]
+[features]
+serde-1 = ["serde"]
+test_debug = []
+test_low_transition_point = []
-- 
2.25.0

